### ISSUE TITLE
sudo make install not working with kernel linux-6.11.1.arch1-1

### ISSUE THREAD

**Issue by homebobhomebob** [siduck/chadwm#197]
To reply this issue: 
I have updated my arcolinux OS (arch rolling) with latest kernel linux 6.11.1-arch1-1, i've made some changes to the chadwm config files, then i go to the terminal and do:

`make clean`
`sudo make install`

but not ends, suddenly close my session to the sddm login screen, and nothing installed.

I tried to downgrade my kernel to linux 6.10.10.arch1-1 and everything is fine, i can install it.

Is a kernel issue? It my bad?

**Comment by siduck** [comment]
i dont get why the kernel should be an issue, i heard a lot of issues with sddm with chadwm,   can you try launching it from tty?

**Comment by MinePro120** [comment]
I have been experiencing the same issue for some days now (I'm using lightdm btw). I will test it tomorrow to see if it's kernel-related.

**Comment by homebobhomebob** [comment]
> i dont get why the kernel should be an issue, i heard a lot of issues with sddm with chadwm, can you try launching it from tty?

Yes, if i change to tty, i can build it. Thanks a lot (I haven't tried that)
I said that may be a kernel issue because downgrading the kernel version, i can build it . (i have not get any issue never before)

Launching from TTY is a valid option for me till next kernel update (hope solved). Thank you
